app/konnect/centralized-consumers.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Centralized Consumers
+alpha: true
+---
+
+A [Consumer](/gateway/latest/key-concepts/consumers/) is a {{site.base_gateway}} entity that consumes or uses the APIs managed by {{site.base_gateway}}. These can be things like applications, services, or users who interact with your APIs. Consumers can be scoped to a {{site.konnect_short_name}} region and managed centrally (centralized Consumers) or be scoped to a control plane in Gateway Manager.
+
+Centralized Consumers provide the following benefits:
+* The Consumer identity can be setup centrally instead of being defined across multiple control planes.
+* Share Consumers across multiple control planes. Users don't need to replicate changes to Consumer identity in multiple control planes and Consumer configuration doesn't conflict.
+* Reduces configuration sync issues between the control plane and the dataplane. Centralized Consumers aren't part of the configuration that is pushed down from the control plane to the dataplane, so it reduces config size and latency. 
+
+## How Centralized Consumers work
+
+When you create a centralized Consumer, you must assign it to a realm. A realm groups centralized Consumers around an identity, defined by organizational boundaries, such as a production realm or a development realm. Realms are connected to a [geographic region](/konnect/geo/) in {{site.konnect_short_name}}.
+
+The following diagram shows how you can configure centralized Consumers in realms as well as scope Consumers to a Control Plane:
+
+<!--vale off -->
```

Comment:
I don't really understand what this diagram is showing me. 
For example:

> Consumer: Ira
> Consumer Groups: Gold for appointments

Does this consumer only have access to the appointments control plane? Or does it have access to both control planes, but since the consumer group Gold only exists on `appointments`, it can only access appointments? Or something else?

Are control planes located inside realms, or alongside them?